lab-cortney/gulpfile.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
This is a good start, there are a few changes you can make which will improve the file.  Please see the comments below which go through the file line by line. :)